src/meta/proto-conv/tests/it/user_proto_conv.rs

```diff
@@ -156,6 +158,7 @@ pub(crate) fn test_s3_stage_info_v14() -> mt::UserStageInfo {
         copy_options: mt::CopyOptions {
             on_error: mt::OnErrorMode::SkipFileNum(666),
             size_limit: 1038,
+            split_size: 0,
```

Comment:
Please update the version note vector [here](https://github.com/datafuselabs/databend/blob/1852734c653d7d3090cb4efce9ed2e35b8d5cc83/src/meta/proto-conv/src/util.rs#L64), and add new version of tests to affected ProtoBuf structs.